engine/src/input/axis.cpp

```diff
@@ -20,7 +20,8 @@ CUBOS_REFLECT_IMPL(cubos::engine::InputAxis)
         .with(FieldsTrait{}
                   .withField("positive", &InputAxis::mPositive)
                   .withField("negative", &InputAxis::mNegative)
-                  .withField("gamepadAxes", &InputAxis::mGamepadAxes));
+                  .withField("gamepadAxes", &InputAxis::mGamepadAxes)
+                  .withField("mDeadzone", &InputAxis::mDeadzone));
```

Comment:
```suggestion
                  .withField("deadzone", &InputAxis::mDeadzone));
```
This way the JSON is prettier :relieved: